### PR TITLE
Update use-dkim-to-validate-outbound-email.md

### PR DESCRIPTION
**Pull request by puneethmeister**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/office-365-security/use-dkim-to-validate-outbound-email.md [hunk 1]
```diff
@@ -255,7 +255,7 @@ If at some point in the future you decide to add another custom domain and you w
 ## Disabling the DKIM signing policy for a custom domain
 <a name="DisableDKIMSigningPolicy"> </a>
 
-Disabling the signing policy does not completely disable DKIM. After a period of time, Microsoft 365 will automatically apply the default policy for your domain. For more information, see [Default behavior for DKIM and Microsoft 365](use-dkim-to-validate-outbound-email.md#DefaultDKIMbehavior).
+Disabling the signing policy does not completely disable DKIM. After a period of time, Microsoft 365 will automatically apply the default policy for your domain, if deafult policy is still in enabled state. if you wish to completely disbale DKIM, you need to disable DKIM on both custom and default domains. For more information, see [Default behavior for DKIM and Microsoft 365](use-dkim-to-validate-outbound-email.md#DefaultDKIMbehavior).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/use-dkim-to-validate-outbound-email.md]:
```suggestion
Disabling the signing policy does not completely disable DKIM. After a period of time, Microsoft 365 will automatically apply the default policy for your domain, if the default policy is still in the enabled state. If you wish to completely disable DKIM, you need to disable DKIM on both the custom and default domains. For more information, see [Default behavior for DKIM and Microsoft 365](use-dkim-to-validate-outbound-email.md#DefaultDKIMbehavior).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @puneethmeister, kindly commit the suggested changes. Many thanks.

**Comment by puneethmeister** [comment]
@MSFTTracyP Gentle reminder

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by MSFTTracyP — COMMENTED** [review]
Thanks

**Pull request merged**